react-client/react-app/src/api/request.js

```diff
@@ -17,4 +17,6 @@ export default function request(theUrl, method, accessToken, params = null){
         }
         xmlHttp.send(JSON.stringify(params));
     })
+
+    //webby sux
```

Comment:
wtf no i dont, pleazse chamge this dickhead

Reply:
suck my knob